ncm-spma/src/main/perl/spma/yum.pm

```diff
@@ -363,11 +363,20 @@ sub expand_groups
 
     my $pkgs = Set::Scalar->new();
 
-    while (my ($group, $types) = each(%$groups)) {
+    while (my ($group_e, $types) = each(%$groups)) {
+        # A group can contain spaces and thus is escaped
```

Comment:
I'm afraid they are... At least this is my experience! panc considers a key with an embedded space as invalid... I think that internally panc has no problem with spaces in keys but that the restriction was maintained until we removed the support for all the backends which cannot deal with them.